src/agent/v8/inspector-debugapi.ts

```diff
@@ -42,6 +42,17 @@ interface InspectorOptions {
   useWellFormattedUrl: boolean;
 }
 
+/**
+ * In older versions of Node, the script source as seen by the Inspector
+ * backend is wrapped in `require('module').wrapper`, and in new versions
+ * (Node 10.16+, Node 11.11+, Node 12+) it's not. This affects line-1
+ * breakpoints.
+ */
+const USE_MODULE_PREFIX = utils.satisfies(
+  process.version,
+  '<10.16 || >=11 <11.11'
```

Comment:
I basically did the same thing in one of my libraries.

https://github.com/istanbuljs/v8-to-istanbul/blob/master/lib/v8-to-istanbul.js#L11

Reply:
Thanks, I did see that `v8-to-istanbul` was mentioned in the Node PR that made this change, though I didn't realize it was your library 👍